src/map/lapack2flash/FLASH_lapack2flash_util.c

```diff
@@ -245,53 +241,46 @@ int FLAME_QR_piv_preorder( FLA_Obj A, int *jpiv_lapack, int *jpiv_fla )
 }
 */
 
-/*---------------------Get/Set Helper---------------------*/
-
-FLA_Error FLASH_set_preferred_blocksize( dim_t blocksize )
-{
-    flash_blocksize = blocksize;
-
-    return FLA_SUCCESS;
-}
-
-dim_t FLASH_get_preferred_blocksize( void )
+FLA_Bool FLASH_Check_Offload_to_GPU( dim_t blocksize, int m, int n, unsigned int offset )
 {
-    return flash_blocksize;
-}
-
-FLA_Error FLASH_set_n_preferred_threads( unsigned int threads )
-{
-    flash_n_threads = threads;
-
     #ifdef FLA_ENABLE_HIP
-        int device_count = 1;
-        FLASH_Queue_available_devices_hip( &device_count );
+        // 'offset' is the number of tiles where it crosses over from CPU to GPU offloading
+        // Checking to see if the area of m and n is less than X tiles and hip is enabled. If so, disable
+        if ( ( blocksize * blocksize * offset > m * n ) && FLASH_Queue_get_enabled_hip() )
```

Comment:
should this be named "crossover" instead of "offset"? you're the native speaker, your choice.

Reply:
Either makes sense to me, I can change! If I remember, I was trying to figure out a better name for this anyway